catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogPropertiesMeta.java

```diff
@@ -36,6 +36,10 @@ public class HiveCatalogPropertiesMeta extends BaseCatalogPropertiesMetadata {
 
   public static final String FETCH_TIMEOUT_SEC = "kerberos.keytab-fetch-timeout-sec";
 
+  public static final String LIST_ALL_TABLES = "list-all-tables";
+
+  public static final boolean DEFAULT_LIST_ALL_TABLES = true;
```

Comment:
I prefer the default value to be false since this has no performance impact and gives the user a clear result.

Reply:
> I prefer the default value to be false since this has no performance impact and gives the user a clear result.

done